api/topo/topo.proto

```diff
@@ -54,10 +52,12 @@ message ReadResponse {
   repeated Object objects = 1;
 }
 
-message StreamMessageRequest {
+message SubscribeRequest {
+  repeated Reference refs = 1;
```

Comment:
I think you should have a `bool withoutReplay = 2;` here. The user might not want a replay of everything in the store, just updates after the command starts